portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio 1</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap" rel="stylesheet">
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <nav> 
+          <span>Navbar</span>
+            <a href="#box">Color Picker</a>
+            <a href="#gallery">Gallery</a>
+            <a href="#aboutContent">About</a>
+      </nav>
+
+    <div class="hero"> 
+        <div class="hero-text"> 
+            <h1>Sofia Montana</h1>
+            
+        </div>
     </div>
+
+    <div id="aboutContent"> 
```

Comment:
separate words in ids by dashes. This should be about-content.

https://screenshot.googleplex.com/0GqFBCXqUA6

Reply:
I used camel case for most of my naming conventions for this site. I did not know that we use snake case for id I will take a note of that and make changes.